definitions/tools/concordance.cwl

```diff
@@ -33,6 +33,11 @@ inputs:
         inputBinding:
             position: 4
         secondaryFiles: [.bai]
+    bam_3:
+        type: File?
+        inputBinding:
+            position: 5
+        secondaryFiles: [.bai]
```

Comment:
Would it be cleaner to pass this as an array of bams, since it can accept N samples?

Reply:
I was thinking of it too but not sure how secondary files .bai (required for somalier) can be passed along with corresponding bams if type: File[].

Reply:
to make sure they get passed through you should be able to do something like https://github.com/genome/analysis-workflows/blob/bcc6adaf15035f5ce6fc851e27b1173b0fd20c1c/definitions/tools/gatk_genotypegvcfs.cwl#L21-L28

```
type: File[]
```
is a shortcut for 
```
type:
  type: array
  items: File
```